docs/UserGuide.md

```diff
@@ -156,16 +156,65 @@ Format: `exit`
 
 ### Saving the data
 
-AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+ContacX data are saved in the hard disk automatically after any command that changes the data in JSON format at `[JAR file location]/data/contacx.json`.
 
+### Exporting the data: `exportcsv`
+
+Exports the current list as a CSV file that can be imported through ContacX or other contact software such as Google/Outlook
+
+Format: `exportcsv EXPORTTYPE`
+* Exports CSV as per the specified format.
+* If `EXPORTTYPE` is left empty, it will default to `EXPORTTYPE = 1` and export in custom ContacX format
+* `EXPORTTYPE` options are
+   * `EXPORTTYPE=1` for ContacX format
+   * `EXPORTTYPE=2` for Google Contacts format
+   * `EXPORTTYPE=3` for Microsoft Outlook format
+* File will be saved on the directory `[JAR file location]/data/addressbook.csv`
+
+Examples:
+* `exportcsv 2`: Exports the current address book as a CSV file ready to be uploaded onto Google Contacts 
 ### Editing the data file
 
-AddressBook data are saved as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+ContacX data are saved as a JSON file `[JAR file location]/data/contacx.json`. Advanced users are welcome to update data directly by editing that data file.
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-If your changes to the data file makes its format invalid, AddressBook will discard all data and start with an empty data file at the next run.
+If your changes to the data file makes its format invalid, ContacX will discard all data and start with an empty data file at the next run.
 </div>
 
+### Importing data
+
+In order to import data from external sources into ContacX, there are the following options:
+
+* Replacing `contacx.json` file with a valid ContacX formatted generated by another system with ContacX
+* Simple CSV import (GUI-based option)
+* Advanced CSV import (Command based)
+
+### Simple CSV Import
+
+Select the `Import` button under `File->Import` on the top bar.
+
+!['basic import UI'](images/BasicImportUI.png)
+
+Upon click, the default OS file picker should show up, with filters for only .csv and .json files to be accepted. Accessing import via the top menu will allow users to import their task and appointments list as JSON/CSV files that were generated by another instance of ContacX.
```

Comment:
I think this is too specific for a user-facing guide. Consider simplification to
> Upon ~click~ *clicking*, you will be prompted to select `.csv` or `.json` files.

Reply:
ok will fix this